*** tests/A-10-07-M-ok.out	2018-11-28 19:24:28.000000000 +0000
--- tests/A-10-07-M-ok.outhyp	2018-12-06 15:07:43.425572597 +0000
***************
*** 22,33 ****
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
--- 22,35 ----
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Entregar projecto: Operação inválida: O projecto 'P2' não existe na disciplina 'D1'.
! Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Mensagem de entrega: Entregar projecto: Operação inválida: O projecto 'P2' não existe na disciplina 'D1'.
! Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
