*** tests/A-10-11-M-ok.out	2018-11-29 10:43:50.000000000 +0000
--- tests/A-10-11-M-ok.outhyp	2018-12-06 15:07:44.005616669 +0000
***************
*** 17,26 ****
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P1
! * 100012 - Entreguei!
! * 100013 - D1-P1 ai2
! Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
--- 17,23 ----
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
